sympy/core/add.py

```diff
@@ -461,7 +461,13 @@ def as_two_terms(self):
         return self.args[0], self._new_rawargs(*self.args[1:])
 
     def as_numer_denom(self):
+        """expression -> a/b -> a, b
```

Comment:
What about making a sentense like `Decomposes an expression to its numerator part and its denominator part.` and take an example to showcase something.

Reply:
Okay, I'll update.